### DISCUSSION TITLE
tutorials/voyage-part3/03-publishing-to-a-graph-variant

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#136] [course feedback]
# tutorials/voyage-part3/03-publishing-to-a-graph-variant

https://www.apollographql.com/tutorials/voyage-part3/03-publishing-to-a-graph-variant

**Comment by rickyssj2** [comment]
Is there a repo that we can clone to get access to the other subgraphs (listings, bookings, payments, etc) to follow along in this course?

**Reply by rickyssj2** [reply]
I am aware that this course is meant to be 'read-only'. But from the images it looks like you have already created the other subgraphs, so it would be nice to have access so that if someone wants to follow along can do so. 

Although, I think creating those subgraphs is planned to be a separate Lab (coming soon). But I am afraid that my free trial of Enterprise will end by then 👀.

**Reply by lizhennessy** [reply]
Hi there @rickyssj2 ! We use this same starter code in [Voyage 2](https://www.apollographql.com/tutorials/voyage-part2/01-airlock-the-monolith), where we explore ways of breaking down a monolithic code base. This contains code for all of the different domains (listings, reviews, bookings, etc.), however they're not yet "federated" as such, and would still need to be broken out into individual subgraphs (this process is also described in Voyage 2.) Alternatively, you can check out our GraphOS series - it doesn't require an Enterprise account, and it covers much of the same material about how to work with GraphOS and ship schema updates safely. Here's the second course in the series which focuses on [Safe Schema Delivery](https://www.apollographql.com/tutorials/graphos-shipping-supergraph)!

**Comment by ejaoua** [comment]
should we publish the staging schema from Voyage II or voyage I ? 
Also in voyage II the accounts schema is not called "accounts.graphql" as stated in the first command to run.
